tools/ts-schema-tool/src/SchemaCommands.ts

```diff
@@ -224,7 +224,15 @@ export class SchemaCommands {
 
     prismSchemaProperties.map(([k, v]) => {
       const nextFieldNode = this.fieldToContext(k, v, objectBuffer, prismSchema);
-      if (v.type === 'object') {
+
+      if (k === 'binary') {
+        // Handle base64 binary fields
+        prismSchema[k] = {
+          "@container": "@set",
+          "@id": "https://schema.org/MediaObject"
```

Comment:
What will be  if this field  have user-set content-uri ?
Maybe we need `if` for this case ?

Reply:
(or comment, which describes - why if is not needed)

Reply:
If contextUri param exists, it will be applied immediately and ignoring this code box. The description was added in `fieldToContext` function. I'm not for sure, we need duplicate it twice. WDYT?